README.md

```diff
@@ -2,4 +2,6 @@
 Just ignore this tbh it's not that important.
 Supposedly?!.
 
-* Mr Croissant 🥐🥐🥐🥐*
\ No newline at end of file
+~~*Mr Croissant 🥐🥐🥐🥐*~~
+
+*🇪🇸🇪🇸🇪🇸🇪🇸 - No ***BODY*** Expects the Spanish Inquisition*
```

Comment:
Fix the Spanish flag - maybe change it to a sword emoji which will actually render:
```suggestion
🗡️🗡️🗡️ *- No **BODY** Expects the Spanish Inquisition*
```